Userland/Libraries/LibWasm/AbstractMachine/Interpreter.cpp

```diff
@@ -707,11 +730,15 @@ void BytecodeInterpreter::interpret(Configuration& configuration, InstructionPoi
     case Instructions::i32_shru.value():
         BINARY_NUMERIC_OPERATION(u32, >>, i32);
     case Instructions::i32_rotl.value():
+        BINARY_PREFIX_NUMERIC_OPERATION(u32, rotl, i32);
     case Instructions::i32_rotr.value():
+        BINARY_PREFIX_NUMERIC_OPERATION(u32, rotr, i32);
     case Instructions::i64_clz.value():
+        UNARY_NUMERIC_OPERATION(i64, __builtin_clzl);
     case Instructions::i64_ctz.value():
+        UNARY_NUMERIC_OPERATION(i64, __builtin_ctzl);
     case Instructions::i64_popcnt.value():
-        goto unimplemented;
+        UNARY_NUMERIC_OPERATION(i64, __builtin_popcountl);
```

Comment:
Shouldn't this use `__builtin_popcountll`?

Reply:
Yea you're right. I'll fix that.